### PR TITLE
upgrade next-translate

### PR DESCRIPTION
**Pull request by osamasayed**
### Summary
Upgrate next-translate package to version `1.0.7`.

### Screenshots

| Before | After |
| ------ | ------ |
| [image here] | [image here] |

### CODE REVIEW

File: i18n.json [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
+  "locales": ["en", "ar"],
+  "defaultLocale": "en",
   "pages": {
     "*": ["common"],
     "/": ["home"],
     "/about": ["about"],
     "/[chapterid]": ["common"]
   }
-}
+}
```
Comment (tifa2UP) [inline i18n.json:10]: nit: new line

File: package.json [hunk 2]
```diff
@@ -107,4 +107,4 @@
   "lint-staged": {
     "*.{tsx,ts}": "yarn lint:fix"
   }
-}
+}
```
Comment (tifa2UP) [inline package.json:110]: nit: new line

**Review by tifa2UP — CHANGES_REQUESTED** [review]
Looks good masha'Allah. Another part that needs change is migrating the `next-translate/Link` to `next/link`

Afterwards the PR is good to go insha'Allah

**Comment by osamasayed** [comment]
> Looks good masha'Allah. Another part that needs change is migrating the `next-translate/Link` to `next/link`
> 
> Afterwards the PR is good to go insha'Allah

This has already been changed in this PR: https://github.com/quran/quran.com-frontend-next/pull/186/commits/5772491522ca313f052918e4ae9dcbfd49aa4c3e#diff-30469af39e44cfa66c6d1d643f6e1b6520d4d057b6856a43e36fbb3e98e70ea4

File: i18n.json (hunk 1 above)
Comment (osamasayed) [reply i18n.json]: Done

File: package.json (hunk 2 above)
Comment (osamasayed) [reply package.json]: Done

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**